modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -58,8 +58,28 @@ public void testExportImportObjectDefinition() throws Exception {
 			objectDefinition.getExternalReferenceCode(), "AccountEntry");
 
 		testExportImport(
-			"test-object-definition.json", "test-object-definition.json", null,
-			"TestObjectDefinition");
+			"test-object-definition1.json", "test-object-definition1.json",
+			null, "TestObjectDefinition1");
+
+		testExportImport(
+			"test-object-definition-2.root-object-definition.json",
+			"test-object-definition-2.root-object-definition.json", null,
+			"TestObjectDefinition2");
+
+		testExportImport(
+			"test-object-definition-3.bound-object-definition.json",
+			"test-object-definition-3.bound-object-definition.json", null,
+			"TestObjectDefinition3");
+	}
+
+	@Test
+	public void testInvalidImportObjectDefinition() throws Exception {
```

Comment:
@G-Sa in the `BoundObjectDefinitionsExportImportTest` class, you added the `testFailedImport` to the `testExportImportBoundObjectDefinitions` method (which corresponds to the `testExportImportObjectDefinition` method here). 

So why do we need this `testInvalidImportObjectDefinition` method here? I think we should follow **only one** pattern.